### DISCUSSION TITLE
Bem vindo a seção de discussões sobre o projeto informativo sobre a vacina para Covid!

### DISCUSSION THREAD

**Discussion by alexanmtz** [alexanmtz/informativo-vacina-covid-19#1] [General]
## 👋 Bem vindo!
 Estamos usando o novo sistema de discussão do Github para conectar com membros da comunidade. Aqui você pode:
  * Fazer perguntas pertinentes que possam entrar no projeto
  * Compartilhar idéias
  * Se engajar com os membros da comunidade

**Comment by vyk1** [comment]
Uma vez que o vírus muta (como aconteceu), não se perde a vacina?
